templates/palette.tera

```diff
@@ -0,0 +1,156 @@
+---
+whiskers:
+  version: 2.3.0
+  matrix:
+    - flavor
+    - accent
+  filename: "dist/Catppuccin-{{ flavor.identifier | capitalize }}-{{ accent | capitalize }}/theme.colors"
+---
+{%- macro rgb(c) -%}
+{{ c.rgb.r }}, {{ c.rgb.g }}, {{ c.rgb.b }}
+{%- endmacro -%}
+{%- set accent = flavor.colors[accent] -%}
+[ColorEffects:Disabled]
+Color={{ self::rgb(c=base) }}
+ColorAmount=0.30000000000000004
+ColorEffect=2
+ContrastAmount=0.1
+ContrastEffect=0
+IntensityAmount=-1
+IntensityEffect=0
+
+[ColorEffects:Inactive]
+ChangeSelectionColor=true
+Color={{ self::rgb(c=base) }}
+ColorAmount=0.5
+ColorEffect=3
+ContrastAmount=0
+ContrastEffect=0
+Enable=true
+IntensityAmount=0
+IntensityEffect=0
+
+[Colors:Button]
+BackgroundAlternate={{ self::rgb(c=accent) }}
+BackgroundNormal={{ self::rgb(c=surface0) }}
+DecorationFocus={{ self::rgb(c=accent) }}
+DecorationHover={{ self::rgb(c=surface0) }}
+ForegroundActive={{ self::rgb(c=peach) }}
+ForegroundInactive={{ self::rgb(c=subtext0) }}
+ForegroundLink={{ self::rgb(c=accent) }}
+ForegroundNegative={{ self::rgb(c=red) }}
+ForegroundNeutral={{ self::rgb(c=yellow) }}
+ForegroundNormal={{ self::rgb(c=text) }}
+ForegroundPositive={{ self::rgb(c=green) }}
+ForegroundVisited={{ self::rgb(c=mauve) }}
```

Comment:
```suggestion
BackgroundAlternate={{ self::rgb(c=accent) }}
BackgroundNormal={{ self::rgb(c=surface0) }}
DecorationFocus={{ self::rgb(c=accent) }}
DecorationHover={{ self::rgb(c=surface0) }}
ForegroundActive={{ self::rgb(c=peach) }}
ForegroundInactive={{ self::rgb(c=subtext0) }}
ForegroundLink={{ self::rgb(c=accent) }}
ForegroundNegative={{ self::rgb(c=red) }}
ForegroundNeutral={{ self::rgb(c=yellow) }}
ForegroundNormal={{ self::rgb(c=text) }}
ForegroundPositive={{ self::rgb(c=green) }}
ForegroundVisited={{ self::rgb(c=mauve) }}
```

Might be overcomplicating this more than it simplifies, but seeing as these sections are all the same you could abstract them into a `{{ self::section(colors=[accent, surface0, ...] }}`

Reply:
It's a draft, I'm not looking for feedback at this time.

Reply:
Well aware, was just curious and wanted to mention it.